### ISSUE TITLE
I need help

### ISSUE THREAD

**Issue by sailnorth** [fjh658/IDA7.0_SP#4]
sorry，I don‘t know how fix this bug，could you please help me step by step？thx！

**Comment by thinkycx** [comment]
## step one
git clone https://github.com/fjh658/IDA7.0_SP.git
cd IDA7.0_SP
## step two
cp libqcocoa.dylib /Applications/IDA Pro 7.0/ida.app/Contents/PlugIns/platforms/libqcocoa.dylib

**Comment by fjh658** [comment]
Are you sure that there is only one file libqcocoa.dylib.bak about the platforms directory? 

You need to ensure that the backup file libqcocoa.dylib does not end with **dylib**.

@sailnorth @thinkycx

**Comment by thinkycx** [comment]
Yes. Before I cp the libqcocoa.dylib to the platforms directory, there is no other files in it.
![image](https://user-images.githubusercontent.com/13138128/48981054-c6614d80-f10b-11e8-86e3-16bc1b98848b.png)

**Comment by fjh658** [comment]
Checking binary file checksum:
```
md5 libqcocoa.dylib
MD5 (libqcocoa.dylib) = 9c8fa1ec2d16bc18e326f362918b0cb9

shasum libqcocoa.dylib
23d94e8dae902515f1587e4bda8292e536c8e25a  libqcocoa.dylib
```

**Comment by thinkycx** [comment]
> Checking binary file checksum:
> 
> ```
> md5 libqcocoa.dylib
> MD5 (libqcocoa.dylib) = 9c8fa1ec2d16bc18e326f362918b0cb9
> 
> shasum libqcocoa.dylib
> 23d94e8dae902515f1587e4bda8292e536c8e25a  libqcocoa.dylib
> ```
> @thinkycx

Wow....it's different!
```
➜  platforms md5 libqcocoa.dylib
MD5 (libqcocoa.dylib) = ff8a1f3fea897c40055cefd7bb7ccf40
➜  platforms
➜  platforms shasum libqcocoa.dylib
e1644a89b27aaf570cdb716f72d5e50612373cbf  libqcocoa.dylib
➜  platforms

```

**Comment by fjh658** [comment]
You should know how to do it. @thinkycx

**Comment by thinkycx** [comment]
It's very interesing. I remember I have copied it before. And I can find my copy history .
But my IDA doesn't crash now.
@fjh658

**Comment by fjh658** [comment]
Please using the libqcocoa.dylib of IDA7.0_SP's master.


- Fixed when multiply input method the IDA pro7.0 on mojave, ida will crash in non-english input method.
- Fixed shortcuts do not work in non-english input method. Eg: F2, tab, ctrl+enter etc.
- Added load bundle for open dialog (The official Qt does not support this feature, but this is suitable for ida )
@thinkycx

**Comment by thinkycx** [comment]
I got the point. 

You have updated the binary 21days ago. 
However, I did't update because I don't know the updates information.
What are the differences between them? (you can see the picture below)

![image](https://user-images.githubusercontent.com/13138128/48981206-8dc27380-f10d-11e8-8d7c-c2be07d67317.png)
@fjh658

**Comment by sailnorth** [comment]
thx a lot @thinkycx @fjh658 i learn a lot from you

**Comment by sailnorth** [comment]
> Yes. Before I cp the libqcocoa.dylib to the platforms directory, there is no other file in it.
> ![image](https://user-images.githubusercontent.com/13138128/48981064-e6910c80-f10b-11e8-936c-7c43a94770ad.png)
> 
> ```
> ➜  platforms pwd
> /Applications/IDA Pro 7.0/ida64.app/Contents/PlugIns/platforms
> ➜  platforms ls
> libqcocoa.dylib     libqcocoa.dylib.bak
> ➜  platforms
> ```
> @fjh658

i wanna know what’s your fantastic terminal‘s name，thx

**Comment by fjh658** [comment]
Added load bundle(package) for **open dialog** that **.app** as directory. @sailnorth 

![image](https://user-images.githubusercontent.com/5550316/48981275-3e307780-f10e-11e8-980f-8ab1d415f2a1.png)

**Comment by fjh658** [comment]
iTerm2   @sailnorth

**Comment by thinkycx** [comment]
> > Yes. Before I cp the libqcocoa.dylib to the platforms directory, there is no other file in it.
> > ![image](https://user-images.githubusercontent.com/13138128/48981064-e6910c80-f10b-11e8-936c-7c43a94770ad.png)
> > ```
> > ➜  platforms pwd
> > /Applications/IDA Pro 7.0/ida64.app/Contents/PlugIns/platforms
> > ➜  platforms ls
> > libqcocoa.dylib     libqcocoa.dylib.bak
> > ➜  platforms
> > ```
> > @fjh658
> 
> i wanna know what’s your fantastic terminal‘s name，thx
## About my terminal
Yes. iTerm2 + zsh + oh-my-zsh and some plugins on it.
You can install it with one script I wrote.  https://gist.github.com/thinkycx/2e21c3572a8d1fde21aad07a58fcf940
@fjh658 @sailnorth

**Comment by sailnorth** [comment]
thx a lot！！！！！！ @thinkycx @fjh658

**Comment by thinkycx** [comment]
I saw the updates here.
https://github.com/fjh658/IDA7.0_SP/commits/master.
 I have updated the binary to the latest. Thank you. @fjh658  
```
➜  platforms shasum libqcocoa.dylib
23d94e8dae902515f1587e4bda8292e536c8e25a  libqcocoa.dylib
➜  platforms md5 libqcocoa.dylib
MD5 (libqcocoa.dylib) = 9c8fa1ec2d16bc18e326f362918b0cb9
```
![image](https://user-images.githubusercontent.com/13138128/48981400-e98dfc00-f10f-11e8-8548-4e2584739646.png)

**Comment by fjh658** [comment]
👍 @thinkycx
